platforms/network-schema.json

```diff
@@ -123,8 +123,8 @@
             "type": "object",
             "properties": {
               "url": { "type": "string","pattern": "^(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9](?::[0-9]{2,5})?(?:\\/[a-z0-9-]{1,61})*$", "description": "Docker registry url. Must be private registry for Corda Enterprise"},
-              "username": { "type": "string","pattern":"^[a-zA-Z0-9._]+$","description": "Username credential required for login."},
-              "password": {"type": "string","description": "Password credential required for login."}
+              "username": { "type": "string","pattern":"^[a-zA-Z0-9.-]+$","description": "Username credential required for login."},
+              "password": {"type": "string","pattern":"^[a-zA-Z0-9._]+$","description": "Password credential required for login."}
```

Comment:
Why is password pattern matching required ? and for username the regex will not allow underscore now